### DISCUSSION TITLE
Rather slow. Am I doing something wrong?

### DISCUSSION THREAD

**Discussion by JohnKirk1337** [ogkalu2/comic-translate#75] [Q&A]
On Japanese to English with the default OCR it's takes hours for 100 pages. The CPU usage is at 50% and the gpu is only about 15~ and It's barely using any vram. The part it hangs on the longest is "Forming Text Blocks." The use GPU box is checked. Did I fuck up something, or is there something I can do to speed this up?

OS:

- Edition Windows 11 Pro
- Version 24H2
- Installed on ‎16/‎06/‎2024
- OS build 26100.1000
- Experience Windows Feature Experience Pack 1000.26100.6.0

Hardware:

- Processor AMD Ryzen 5 5600G
- Installed RAM 32,0 GB
- GPU NVIDIA GeForce RTX 3060

**Comment by ogkalu2** [comment]
I'll look into it but i don't think you're doing anything wrong. I don't personally use the default OCR options so i don't know how slow or not they should be. How long does one image take to process ?

**Reply by JohnKirk1337** [reply]
Anywhere from 30 seconds to 2 minutes, depending on the image. 80% of the time is spent on forming blocks. What irks me is that it's barely using the GPU. Shouldn't it be doing more images in parallel? I've started running 3 instances instead.
